services/net/content/ContentManager.cs

```diff
@@ -256,6 +256,7 @@ private async Task HandleMessageAsync(ConsumeResult<string, SourceContent> resul
                 Body = !String.IsNullOrWhiteSpace(model.Body) ? StringExtensions.SanitizeContent(model.Body, "<p(?:\\s[^>]*)?>|</p>", Environment.NewLine) : model.ContentType == ContentType.Snippet ? "" : StringExtensions.SanitizeContent(model.Summary, "<p(?:\\s[^>]*)?>|</p>", Environment.NewLine),
                 SourceUrl = model.Link,
                 PublishedOn = model.PublishedOn,
+                Byline = model.Authors.FirstOrDefault()?.Name ?? ""
```

Comment:
Change to `Byline = String.Join(",", model.Authors.Select(a => a.Name))`

Reply:
Updated, thanks